### DISCUSSION TITLE
[Feedback Needed] Compare different experiment result in WebUI

### DISCUSSION THREAD

**Discussion by xuehui1991** [microsoft/nni#3791] [New Feature Design Discussion]
This is a **help wanted** to all the NNI users. If you have any suggestions or scenarios related to the following question, please add a comment in the issue. 

**What would you like to be added**:
Do we need to add a function that compares different experiments result in WebUI? like  "nnictl compare --expeirment_id1 id1 --expeirmnet_id2 id2"?

**Why is this needed**:

**Without this feature, how does current nni work**：

**Comment by scarlett2018** [comment]
@kostasfilip - would you like to share more suggestions given you had raised similar request in #774 ? thanks.

**Comment by rabbit008** [comment]
1.Compare different tuners,searchspace by default metric graph.

**Comment by rabbit008** [comment]
NNI v1.1 will add a new command, 'nnictl view expriment_id' to view the webui of the experiment that has been stopped without resume it. When you specifies a different port number, you can open different experimental webui at the same time for comparison.But this may be a bit of a hassle, if you have specific comparison needs, please add a comment in this issue.

**Comment by ultmaster** [comment]
Our current design is to add a command in nnictl that will compare several experiments in the past and generate, maybe, a simple HTML.

**Comment by scarlett2018** [comment]
@kvartet - may you help to link the "NNI Project" design here? thanks.

**Reply by kvartet** [reply]
linked, see [WebUI improvement Project](https://github.com/microsoft/nni/projects/3).

**Reply by scarlett2018** [reply]
> linked, see [WebUI improvement Project](https://github.com/microsoft/nni/projects/3).

@kvartet the link does not work

**Reply by Lijiaoa** [reply]
design: https://www.figma.com/file/T7HTc67K4dmyUBS9cnNpBJ/0427-Full-process?node-id=0%3A1

**Reply by kvartet** [reply]
Discussions cannot be linked to the Project, so I add an item in the WebUI Project, see below.
![image](https://user-images.githubusercontent.com/48014605/121667199-9bc61d00-cadc-11eb-9208-ab64f2834272.png)
